librecores-ci/README.md

```diff
@@ -25,3 +26,49 @@ The [or1k Continuous Integration (CI) suite](https://github.com/openrisc/or1k-te
 Librecores-CI docker container in [Travis-CI](https://github.com/openrisc/mor1kx/blob/master/.travis.yml). 
 Parallel execution of tests runs in Librecores-CI docker environment. As a reference one can follow up the [blog](http://nancychauhan.in/stories/2019/06/08/gsoc-week1_2/) to get an idea of how librecores docker images can be 
 integrated to existing continuous integration suite. 
+
+#### Yosys Parser
+Yosys is a framework for Verilog RTL synthesis. For monitoring resource usages of any hardware project, this parser script (extract-yosys-stats.py) can help to visualise results better. This take input from huge yosys.log file and outputs 'Printing Statistics' like :
+```
+wire bits,16199
+public wires,1058
+public wire bits,11151
+memories,0
+memory bits,0
+processes,0
+cells,7310
+SB_CARRY,426
+SB_DFF,166
+SB_DFFE,885
+SB_DFFESR,569
+SB_DFFESS,13
+SB_DFFSR,56
+SB_DFFSS,2
+SB_LUT4,5182
+SB_RAM40_4K,11
+```
+
+#### Quick Start 
+
+To use Yosys Parser, you can see its implementation in <a href="https://github.com/openrisc/mor1kx">openrisc project<a>.
+
+One can quickly get started with Fusesoc, a package manager and a set of build tools for HDL (Hardware Description Language) code.
+Fusesoc provides the <a href = "https://github.com/olofk/edalize">icestorm backend</a> ( edalize ) which uses yosys to handle synthesis, arachne-pnr for place & route and icepack for creating the bitstream.
```

Comment:
Its probably better to use a markdown style link here.

Reply:
You can mention:
    - ...bitstream; in our case we set `pnr: none` as we are only interested in synthesis results.